src/accelerate/commands/launch.py

```diff
@@ -127,10 +128,13 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 
 def launch_command_parser(subparsers=None):
+    description = "Launch a python script in a distributed scenario. Arguments can be passed in with either hyphens (`a-b`) or underscores (`a_b`)"
```

Comment:
I think the doc would be clearer with a real example, for `num-processes` instead of `a-b`.